gitlab/resource_gitlab_instance_cluster.go

```diff
@@ -0,0 +1,248 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabInstanceCluster() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabInstanceClusterCreate,
+		Read:   resourceGitlabInstanceClusterRead,
+		Update: resourceGitlabInstanceClusterUpdate,
+		Delete: resourceGitlabInstanceClusterDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"domain": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			"managed": {
```

Comment:
Is it possible to change `managed` after creation? If not it should be `ForceNew: true`.